### DISCUSSION TITLE
how to allow kitty.app to reflect changes instead of just kitty-wrapped?

### DISCUSSION THREAD

**Discussion by thleo** [kovidgoyal/kitty#3380] [Q&A] [answered]
Install method: nix  
Issue: kitty.conf preferences only display changes from default config when kitty is launched as `.kitty-wrapped` [`kitty` issued from within `kitty.app`]

Within the kitty.app terminal, the config preferences are picked up but no changes are reflected in the terminal appearance.

**Comment by Luflosi** [comment, marked as answer]
kitty doesn't support reloading the config file without restarting it. Did you restart kitty?
Also, if you just want to change certain things, like the colors, you can change them without a restart by using the remote control feature.

**Reply by thleo** [reply]
I thought I had restarted kitty and running `--congif-debug` indicates that the changes to the default config file are recognized, though not reflected in kitty.

Checking this morning, I failed to quit the application (merely closed out all windows) and now see the changes reflected in both kitty.app and wrapper.
